tests/charm++/varTRAM/Makefile

```diff
@@ -0,0 +1,24 @@
+-include ../../common.mk
+-include ../../../include/conv-mach-opt.mak
+CHARMC = ../../../bin/charmc $(OPTS)
+
+OBJS = vartest.o
+
+all: vartest
+
+vartest: vartest.o
+	$(CHARMC) $(CHARMCFLAGS) -language charm++ -o vartest vartest.o -module NDMeshStreamer
```

Comment:
Is it necessary to link with '-module NDMeshStreamer' explicitly? I thought that was built in to charmc/charmxi when using [aggregate]

Reply:
This was a mistake that has since been fixed.